rom/laghos_rom.cpp

```diff
@@ -96,21 +182,6 @@ void ROM_Sampler::SampleSolution(const double t, const double dt, const double p
                 tSnapV.push_back(t);
             }
         }
-
```

Comment:
Why is this case being removed? It is not related to the DMD PR, right? Is it because the case `!sns` is never used?

Reply:
Thanks for pointing that out. It was not intentional.